### PR TITLE
fix: MD019/no-multiple-space-atx

### PR DESCRIPTION
**Pull request by nschonni**
Multiple spaces after hash on atx style heading
Find: ^(#+) ( +)(.*)\n
Repalace:
Filter: $1 $3\n

### CODE REVIEW

**Comment by PRMerger20** [comment]
@nschonni : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

File: docs/ide/editorconfig-code-style-settings-reference.md [hunk 1]
```diff
@@ -2197,9 +2197,7 @@ To help you get started, here is an example *.editorconfig* file with the defaul
 ```EditorConfig
 ###############################
 # Core EditorConfig Options   #
-###############################
-
-root = true
+############################### root = true
```
Comment (nschonni) [inline docs/ide/editorconfig-code-style-settings-reference.md]: This looks off, should have used literal space instead of `\s` I think

**Comment by PRMerger6** [comment]
@nschonni : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

File: docs/ide/editorconfig-code-style-settings-reference.md (hunk 1 above)
Comment (nschonni) [reply docs/ide/editorconfig-code-style-settings-reference.md]: Updated the regex and rebased

**Comment by gewarren** [comment]
It might be a good idea to explicitly look for `<a` after the multiple spaces and fix those in one commit. Then use your original regex to fix the remaining "extra spaces after hash" in a separate commit that would be easier to diff.

**Comment by PRMerger9** [comment]
@nschonni : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

**Comment by nschonni** [comment]
@gewarren I've split the two regexes into separate commits

**Comment by gewarren** [comment]
@nschonni Thanks, looks good!

**Pull request merged**